packages/tokens/src/tokens/color/component-dark.json

```diff
@@ -0,0 +1,9 @@
+{
+  "button-primary-background-on-dark": {
+    "name": "button-primary-background-on-dark",
+    "value": "{semantic-test-on-dark}",
```

Comment:
Is it correct to assume these are just for testing as of this ticket and will no longer have `test` in there with the follow-up tickets?

Reply:
Yes. Anything in the semantic or component json files will be replaced with non-test tokens in the follow up tickets listed above.